storage/engine/rocksdb/encoding.cc

```diff
@@ -61,6 +61,48 @@ bool DecodeUvarint(rocksdb::Slice* buf, T* value) {
 
 }  // namespace
 
+void EncodeBytes(std::string* buf, const char* ptr, int n) {
+  buf->push_back(kBytesMarker);
+  const char* end = ptr + n;
+  for (; ptr != end;) {
+    const char* tmp = reinterpret_cast<const char*>(memchr(ptr, kEscape, end - ptr));
```

Comment:
You can use a `static_cast` instead of `reinterpret_cast` for casting from `void*`, right? (since `kEscape` is 0 you could also find it with `strlen` but that may be getting too clever).